GameData/RealismOverhaul/Waterfall_Configs/_Processor/90_controllers.cfg

```diff
@@ -0,0 +1,72 @@
+// Add controllers to the generated module. Place template-specific controllers first; modules will
+// otherwise receive a default set of controllers.
+
+@PART[*]:HAS[@ROWaterfall]:FOR[zROWaterfall_90_Controllers]:NEEDS[Waterfall]
+{
+    @MODULE[ModuleWaterfallFX]:HAS[@TEMPLATE:HAS[#__rowaterfall[mainPlume],#templateName[waterfall-ntr-lh2-1]]],*
```

Comment:
Sometimes I prefer adding an extraneous `,*` for the reader, because I don't think it's immediately obvious to someone not familiar with ModuleManager that `:HAS` implies "apply to all."

However, in this case, this is a vestige of an older attempt at plume variant switching, so I'll remove them.